src/main/js/controls/Gantt/Track.js

```diff
@@ -220,6 +220,7 @@ class Track extends GraphContent {
      * @inheritdoc
      */
     resize(graph) {
+        scaleGraph(graph.scale, graph.config);
```

Comment:
Any unit tests for the change?

Reply:
No veera. Test cases regarding scaling is already written veera.

Reply:
Should the test cases be updated to ensure this bug does not occur?

Reply:
Added test cases to check y-co ordinate position and graph scale domain after unloading track. https://github.com/cerner/carbon-graphs/pull/263/commits/915542d71c0d77ba712b18fe3bef681f8dd79878